estela-web/src/utils.ts

```diff
@@ -39,7 +39,7 @@ export function setValArr({ arr, val, index }: { arr: number[]; val: number; ind
 
 export function parseDuration(duration: string | undefined): Duration {
     if (duration && duration !== "undefined") {
-        const durationRegex = /(?:(\d+) days?,\s*)?(\d{1,2}):(\d{1,2}):(\d{1,2}?\.*(\d*))?/;
+        const durationRegex = /(?:(\d+) \s*)?(\d{1,2}):(\d{1,2}):(\d{1,2}?\.*(\d*))?/;
```

Comment:
Django uses timedelta for the DurationField (https://docs.djangoproject.com/en/4.2/ref/models/fields/#durationfield) when calling the str() this method returns: "[D day[s], ][H]H:MM:SS[.UUUUUU]". Uhmm Have you tested it correctly? if yes, please forget this comment.

Reply:
Yes, when calling the `str` method from inside Django, that's how it's returned. However, the serializer returns it as `D HH:MM:SS`. I have tested this thoroughly 👍🏻